components/homekit/__init__.py

```
@@ -1,7 +1,7 @@
 from esphome import automation
 import esphome.codegen as cg
 import esphome.config_validation as cv
-from esphome.components import mdns, wifi, light, lock, sensor, switch, climate, pn532
+from esphome.components import mdns, wifi, light, lock, sensor, switch, climate, pn532, fan
```

Ruff (0.8.2)

`esphome.components.mdns` imported but unused

(F401)